grab_all_data.py

```diff
@@ -0,0 +1,138 @@
+"""
+Get CSV Data : https://www.cryptodatadownload.com/data/binance/
+"""
+
+import sqlite3
+import time
+from dataframe import GetDataframe
+import pickle
+import keyboard
+
+total_years = 2
+months = 12 * total_years
+days = 30 * months
+hours = 24 * days
+minute = hours * 60
+print(f"We are grabbing '{minute}' candles")
+# print(input("Find Minutes:"))
+time_of_data = int(minute)
+
+# Time Counting
+StartTime = time.time()
+print("This Script Start " + time.ctime())
+
+from get_symbol.find_symbols import FindSymbols
+import pandas as pd
+pd.set_option('mode.chained_assignment', None)
+#
+# pd.set_option('display.max_rows', 500)
+# pd.set_option('display.max_columns', 500)
+# pd.set_option('display.width', 1000)
+#
+from api_callling.api_calling import APICall
+ticker_info = pd.DataFrame(APICall.client.get_ticker())
+# print(ticker_info)
+fs = FindSymbols()
+all_symbols_payers = fs.get_all_symbols("BUSD", ticker_info)['symbol']
+
+# print(all_symbols_payers)
+# symbol_data_already_collected
+
+try:
+    with open('symbol_data_already_collected.pkl', 'rb') as f:
+        symbol_data_already_collected = pickle.load(f)
+    print("symbol list loaded from file")
+except FileNotFoundError:
+    print("symbol file not found, creating new list.")
+    symbol_data_already_collected = []
+
+print(symbol_data_already_collected)
+
+# print(input(":"))
+
+for symbol in all_symbols_payers:
+
+    # if press q then the code will be stop here
+    if keyboard.is_pressed('q'):
+        break
+
+    if symbol in symbol_data_already_collected:
```

Comment:
Use Sql data to see already downloaded data.

Reply:
We will do this thing on the missing data script.